core/config.go

```diff
@@ -354,6 +354,9 @@ func (c *ChainConfig) GetFeature(num *big.Int, id string) (*ForkFeature, *Fork,
 	var found = false
 	if num != nil && id != "" {
 		for _, f := range c.Forks {
+			if f.Block == nil {
```

Comment:
Just another spin-off from debugging the benchmark test panic. Can't hurt to be safe here, though never happen.